dualscreeninfo/android/src/main/kotlin/com/microsoft/reactnativedualscreen/dualscreen/DualScreenInfo.kt

```diff
@@ -113,6 +114,11 @@ class DualScreenInfo constructor(context: ReactApplicationContext) : ReactContex
 		return "landscapeFlipped"
 	}
 
+	private fun convertPixelsToDp(px: Int): Double {
+		val metrics = reactApplicationContext.resources.displayMetrics
+		return (px / (metrics.density)).toDouble()
```

Comment:
>px / (metrics.density) [](start = 10, length = 22)

This line of code looks like it's doing integer division.  Do we need to convert px and metrics.density to doubles before dividing?

Reply:
you are correct it actually is doing a px(int)/ density(float), i think for safety anyways i can convert the px to a double then divide, I don't think we need to worry about casting the float

Reply:
looks good thanks!

---
In reply to: [456675801](https://github.com/microsoft/react-native-dualscreen/pull/46#discussion_r456675801) [](ancestors = 456675801)